examples/basic/.tina/schema.ts

```diff
@@ -11,7 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { defineSchema } from '@tinacms/cli'
+import { defineSchema } from 'tinacms'
```

Comment:
Does this actually work in the monorepo? I would expect we'd need the `pnpPlugin` for esbuild to know where to find `tinacms`?

Reply:
It was working for me.

Reply:
Strike that, looks like it's been fixed in esbuild https://github.com/evanw/esbuild/blob/master/CHANGELOG.md#0131. Awesome! 👍

Reply:
My only hesitation here is that issues _like_ the PNP one will arise at this point, even though that specific one has been fixed. We'll want to test this with pnpm and npm obviously. I guess at the end of the day if this doesn't work we can tell users to just `'@tinacms/cli'` while we make any necessary fixes. Will these compile errors be reported to our telemetrics system?

Reply:
Yeah I share that concern, I think this will be fine for _most_ use cases. We can leave it in the CLI for these cases.

> Will these compile errors be reported to our telemetrics system?

I think so, let me check. If they are not part of it they should be.